wandb/sdk/launch/runner/sagemaker_runner.py

```diff
@@ -187,6 +188,9 @@ def run(
                 launch_project,
                 self._api,
                 role_arn,
+                launch_project.override_entrypoint,
+                launch_project.override_args,
+                MAX_ENV_LENGTHS[self._type],
```

Comment:
we could just use `self.__class__.__name__`, might make the env dict more robust